app/views/application/head/_favicons.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= favicon_link_tag asset_path("/favicon.svg"), rel: 'icon', type: 'image/svg+xml' %>
+<%= favicon_link_tag asset_path("/favicon.png"), rel: 'icon', type: 'image/png' %>
+<%= favicon_link_tag "/apple-touch-icon.png", rel: 'apple-touch-icon' %>
```

Comment:
If you look at the deleted code back in the _head file, we previously were serving a bunch of different sized apple_touch_icons. Did you mean to remove all of those in favor of this single one? I'm inclined to delete this single one and go back to the whole set of ones within the asset_path("apple_touch_icons/*") but I'm not sure so I didn't make that change.

Reply:
The multiple icons seemed ultimately unnecessary, but harmless if you prefer.

<!-- From Kagi -->
Based on the available information, it seems that you generally do not need multiple Apple touch icons for your web app, and a single icon can often suffice. Here are the key points:

- The W3C specification for Progressive Web Apps (PWAs) does not mandate any specific icon sizes, but allows you to provide multiple icons of various sizes and formats to allow the browser to choose the most appropriate one. [1]
- Many sources suggest that providing a single apple-touch-icon.png file in the root directory of your website is often enough, as modern browsers will automatically scale it to the appropriate size. [2]
- Some sources mention that you can provide multiple icon sizes (e.g. 57x57, 72x72, 114x114 pixels) to cater to different Apple device types, but this is not strictly required. [3][4]
- Overall, the consensus seems to be that a single, well-designed apple-touch-icon.png file is sufficient in most cases, and creating multiple icon files is not necessary. [5][6]
- So in summary, while you can provide multiple touch icons if you wish, a single, high-quality apple-touch-icon.png file is generally recommended and should work well for most web apps. [5][6]

Reply:
I just pushed a commit that corrects the asset_path() so it's back to how you had it, and I went ahead and moved all these files into /public and cleaned up some old files we weren't using. This is all in the commit I just pushed so we should now be good on this.